src/opcode.rs

```diff
@@ -1,14 +1,14 @@
+use crate::eravm_error::EraVmError;
+use crate::eravm_error::OpcodeError;
 use lazy_static::lazy_static;
-use zkevm_opcode_defs::Opcode as Variant;
+pub use zkevm_opcode_defs::Opcode as Variant;
```

Comment:
Was this change necessary for the `zksync-era` integration?

Reply:
Yes, as the libraries used diverge, this way we don't have to add a new crate.